Broken build after merging #757

After merging #757, I'm getting the following error when building `develop`:

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'assembleDebug' not found in project ':blocklydemo'.

* Try:
Run gradle tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

--- a/blocklydemo/build.gradle
+++ b/blocklydemo/build.gradle
@@ -25,6 +25,8 @@ android {
         targetSdkVersion 28
         versionCode 1
         versionName "1.0"
+
+        missingDimensionStrategy "blockly", 'full'
     }
     buildTypes {
         release {
@@ -33,13 +35,6 @@ android {
     }
     lintOptions {
           abortOnError false
-    }
-
-    flavorDimensions 'blockly'
-    productFlavors {
-        full {
-            dimension 'blockly'
-        }
     }
 }
 

